src/drawing/DrawingCreator.tsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import classNames from 'classnames';
 import { bdlBoxBlue } from 'box-ui-elements/es/styles/variables';
+import DrawingCursorTemplate from './DrawingCursor.svg';
```

Comment:
Having this as a separate file will require us (and any eventual consumers of the npm module) to support SVGs in our webpack build. Could the contents of the SVG be stored as a multiline string?